pkg/expr/expr_test.go

```diff
@@ -228,6 +240,111 @@ func TestAssertType(t *testing.T) {
 	}
 }
 
+func TestCacheContent(t *testing.T) {
+	cache, _ := lru.New(1024)
+	cexl := &cexl{
+		fMap: FuncMap(), cache: cache,
+	}
+	str1 := "a.b"
+	_, err := cexl.cacheGetExpression(str1)
+	if err != nil {
+		t.Errorf("Parsing of expression '%s' failed with error '%v'", str1, err)
+		return
+	}
+	expectedEx, _ := Parse(str1)
+
+	exFromCache, _ := cexl.cache.Get(str1)
+	cachedEx := exFromCache.(*Expression)
+
+	if !reflect.DeepEqual(cachedEx, expectedEx) {
+		t.Errorf("Cache contains invalid data: expected=%v, actual=%v", expectedEx, cachedEx)
```

Comment:
switch `expected` and `actual` ordering. see: https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures